www/.env.development

```diff
@@ -0,0 +1 @@
+GATSBY_SCREENSHOT_PLACEHOLDER=true
```

Comment:
We don't want to commit this, unfortunately!

Reply:
oops! sorry! I forgot about this!